packages/babel-traverse/test/scope.js

```diff
@@ -422,7 +422,11 @@ describe("scope", () => {
           }
         });
 
-        /*if (kind1 !== kind2) {
+        if (kind1 !== kind2) {
```

Comment:
👍 for https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-commented-out-tests.md